week1/homework.py

```diff
@@ -0,0 +1,109 @@
+import random
+
+print(' ## Task 1 ## \n')
+#Write a python script to print your name and age
+
+name = "Majid Khan"
+age = int(2021-1983)
+print('My name is ' + name + "\nI am " + str(age) + " years old \n")
+
+
+print(' ## Task 2 ## \n')
+# Create a list of your 5 favorite movies and store it in the variable
+favMovies = ['The Shawshank Redemption', 'The Dark Knight ', 'Catch Me If You Can', 'The Godfather', 'For a Few Dollars More' ]
+for i in range(len(favMovies)):
+    print('My all time favourite movies are; \n', favMovies[i])
```

Comment:
Python can make your work with for loops way easier :)
checkout my last comment here, for how to handle for loops in Python